### ISSUE TITLE
Edge in Defender Application Guard not accessible to NVDA at all

### ISSUE THREAD

**Issue by aursulis** [nvaccess/nvda#15164]
https://github.com/nvaccess/nvda/pull/7600 suggests that NVDA should work with Defender Application Guard (WDAG/MDAG) windows, however the current version produces no speech output when hovering elements or using NVDA-A to say all. As detailed below, I have narrowed this down to it working in 2019.2.1 and being broken from 2019.3 onwards.
### Steps to reproduce:
1. Enable Application Guard through "Turn Windows features on or off", selecting "Microsoft Defender Application Guard", going through the necessary reboots, etc.
2. Start Edge
3. Access the "Settings and More" menu (ALT-F), "New Application Guard window"
4. In the newly appeared Edge window, navigate to http://nvaccess.org and attempt to read the page
### Actual behavior:

NVDA remains silent.
### Expected behavior:

Expect to hear page contents.
### NVDA logs, crash dumps and other attachments:
[nvda.log](https://github.com/nvaccess/nvda/files/12088304/nvda.log) from 2023.1, enabled debug and UIA logging.
#### NVDA installed/portable/running from source:
Tested both installed and running from the installer.
#### NVDA version:
2023.1.0.27913
#### Windows version:
Windows 10 22H2 19045.3208,
Windows 11 22H2 22621.1992
#### Name and version of other software in use when reproducing the issue:
Microsoft Edge 114.0.1823.86
#### Other information about your system:
These are two separate machines running different versions of Windows.
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes. The behaviour works with 2019.2.1 and breaks with 2019.3 onwards.
#### If NVDA add-ons are disabled, is your problem still occurring?
No add-ons are enabled.
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Did not attempt this.

**Comment by amirsol81** [comment]
Still duplicable with 116 Beta 4.

**Comment by Brian1Gaff** [comment]
I notice that there seem to have been a number of reports on parts of Edge 
not being read recently. I wondered if these were connected. Does Narrator 
work?
 Brian

-- 
***@***.***
Sent via blueyonder.(Virgin media)
Please address personal E-mail to:-
***@***.***, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by aursulis** [comment]
Yes, Narrator works. To be clear though, it's only Edge running in Defender Application Guard mode (i.e. a disposable VM) that's affected. Regular Edge is being read fine.

**Comment by CyrilleB79** [comment]
@aursulis has written :
> The behaviour works with 2019.2.1 and breaks with 2019.3 onwards.

and:
> Yes, Narrator works.

@seanbudd, could you explain why you have put blocked/needs-external-fix label? From whom is a fix expected?

**Comment by seanbudd** [comment]
This requires a fix from Windows and has been reported to Microsoft in the past

**Comment by kuchronski** [comment]
> This requires a fix from Windows and has been reported to Microsoft in the past
@seanbudd , can you share Microsoft support case # / EEAP UID and some technical details regarding this problem?

**Comment by seanbudd** [comment]
We weren't given a support case number when we reported this. Feel free to also report this issue to Microsoft

**Comment by kuchronski** [comment]
@seanbudd , can you please provide technical details about this problem, I'd like to escalate it with Microsoft, but I need some sort of reference to the case you've raised with them or a description of the issue.

**Comment by seanbudd** [comment]
@kuchronski you don't need technical details to report an issue to Microsoft, the level of information provided here is sufficient. You just need to explain the problem, how it can be reproduced, and what the impact is to you

**Comment by kuchronski** [comment]
> @kuchronski you don't need technical details to report an issue to Microsoft, the level of information provided here is sufficient. You just need to explain the problem, how it can be reproduced, and what the impact is to you

Thanks for your reply @seanbudd. Hmm, our testing suggests that WDAG works fine with version 2019.2.1 but not with 2019.3 and onwards - I don't suppose there is a possibility that some sort of regression was introduced in 2019.3.
